test/JDBC/expected/babel_4328_datetime-before-16_3-vu-prepare.out

```diff
@@ -0,0 +1,74 @@
+create view babel_4328_datetime_v1 as select cast('2020' as datetime)
```

Comment:
@rohit01010 is working on the fixing the `convert` for this so please cross check all the test cases to make sure both of you use all the test cases combined to enhance the test coverage. Refer this PR https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2571